apps/dashboard/app/views/layouts/nav/_group.html.erb

```diff
@@ -1,9 +1,9 @@
-<li class="nav-item dropdown" title="<%= group.title %>">
-  <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+<li class="nav-item dropdown" role="none">
+  <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="menuitem">
```

Comment:
Should the a tag have the `title` attribute?

Reply:
I guess it could. screen-reader reads it the same either way.  The `ul` under this has a title, but I see what you mean - there's no hover over.

I wonder what the value is in a hover-over that has the same text you see on screen? In any case - I'm happy to add it back as it doesn't seem to affect accessibility.